### PR TITLE
Guides

### PR DESCRIPTION
**Pull request by theomarkkuspaul**

### CODE REVIEW

File: app/controllers/guides_controller.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class GuidesController < ApplicationController
+
+  before_action :admin_signed_in?, only: [:create, :destroy]
+
+  def index
+    @guides = Guide.all
+
+    render :index
```
Comment (EdwinRozario) [inline app/controllers/guides_controller.rb]: dont need this line by default that view is rendered for that method

File: app/controllers/guides_controller.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  def new
+    @guide = Guide.new
+
+    render :new
```
Comment (EdwinRozario) [inline app/controllers/guides_controller.rb]: same here

Comment (theomarkkuspaul) [reply app/controllers/guides_controller.rb]: :+

File: app/controllers/guides_controller.rb (hunk 1 above)
Comment (theomarkkuspaul) [reply app/controllers/guides_controller.rb]: 👍

File: app/uploaders/guide_uploader.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+class GuideUploader < CarrierWave::Uploader::Base
```
Comment (EdwinRozario) [inline app/uploaders/guide_uploader.rb]: Is it suppose to be `Downloadable`? I cant remember

Comment (theomarkkuspaul) [reply app/uploaders/guide_uploader.rb]: this is the uploader for the guide files

**Pull request merged**